src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewController.swift

```diff
@@ -15,11 +15,11 @@ class HealthCertificateViewController: UIViewController, UITableViewDataSource,
 		vaccinationValueSetsProvider: VaccinationValueSetsProviding,
 		dismiss: @escaping () -> Void,
 		didTapValidationButton: @escaping () -> Void,
-		didTapDeleteButton: @escaping () -> Void
+		didTapActionSheet: @escaping () -> Void
```

Comment:
```suggestion
		didMoreButton: @escaping () -> Void
```

Just an Idea that would not hint that there is an ActionSheet that has to be used.

Reply:
yeah, good point. I will change it.